windows-icon-handler/common/include/parsec.h

```diff
@@ -0,0 +1,144 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+#include <array>
+#include <memory>
+#include <string_view>
+#include <sstream>
+#include <fstream>
+
+#include <json/json.hpp>
+
+#include <atlbase.h>
+
+// Namespace shorthand
+using json = nlohmann::json;
+
+namespace parsec
+{
+    enum class SyncState {
+        Synced,
+        Refresh,
+        NotSet
+    };
+
+#ifdef _DEBUG
```

Comment:
I think we're better off removing the debug part. It probably won't be used ever again, makes the code harder to read, will cause problems if we compile in debug by mistake, and contains hardcoded absolute paths.